website/docs/cluster-management/profiles.mdx

```diff
@@ -11,77 +11,85 @@ import TierLabel from "../_components/TierLabel";
 The following instructions require you to make minor changes to the content of your own hosted Helm repository.
 :::
 
-When provisioning new clusters it is often useful to install selected software packages to them as part of their bootstrap process. Weave GitOps Enterprise enables this by installing standard Helm charts to the newly provisioned clusters. This feature lowers the ongoing operational overhead and allows for the clusters to be immediately usable after being provisioned. To set this up you need to:
+To put it simply, Profiles are [Helm charts](https://helm.sh/docs/topics/charts/). To create a Profile, you need to add an [annotation](https://helm.sh/docs/topics/charts/#the-chartyaml-file) to a Helm chart.
 
-1. Annotate a Helm chart to make it available for installation
-2. Select which profiles you want installed when creating a cluster
+A very simple Helm chart marked up as a Profile looks like this:
```

Comment:
I thought we were removing this?

We could at least move it to the very bottom of the page, users don't need to create profiles in order to use them?

Reply:
done